test/devices_tests/sensor_expose_loop_test.py

```diff
@@ -203,10 +203,10 @@ async def test_array_sensor_loop(self, value_type, test_payload, test_value):
         )
         await xknx.telegrams.join()
         xknx.knxip_interface.send_telegram.assert_called_with(outgoing_telegram)
-        if isinstance(test_value, float):
-            assert round(expose.resolve_state(), 4) == test_value
-        else:
-            assert expose.resolve_state() == test_value
+        # if isinstance(test_value, float):
+        #     assert round(expose.resolve_state(), 4) == test_value
+        # else:
```

Comment:
consider removing commented out code, git history has it all

Reply:
oops...